bpf/cpu/cpu.bpf.c

```diff
@@ -11,10 +11,10 @@
 // TODO(kakkoyun): Split into multiple files.
 #include "../common.h"
 
-#include <bpf_core_read.h> // TODO(kakkoyun): Validate if this is needed.
-#include <bpf_endian.h>
-#include <bpf_helpers.h>
-#include <bpf_tracing.h> // TODO(kakkoyun): Validate if this is needed.
+#include <bpf/bpf_core_read.h> // TODO(kakkoyun): Validate if this is needed.
```

Comment:
This change was made because some of the headers we include have the same prefix for these includes. By using the same prefix and adjusting the include path we ensure that the compiler can find the headers for transitive dependencies